Update Troubleshooting crawl errors.md (#8622)
cloudflare/cloudflare-docs: content/support/Troubleshooting/General Troubleshooting/Troubleshooting crawl errors.md

Cloudflare allows search engine crawlers and bots. If you observe crawl issues or Cloudflare challenges presented to the search engine crawler or bot, [contact Cloudflare support](https://support.cloudflare.com/hc/articles/200172476) with the information you gather when troubleshooting the crawl errors via the methods outlined in this guide.

___

## Disable Anti-bot modules

> Search engine crawlers' requests, when proxied through Cloudflare, could have blocked by Anti-Bot modules installed on your origin server. Try disabling your anti-bot modules to help prevent blocking from your origin.

Reviewer: ```suggestion
Search engine crawlers' requests, when proxied through Cloudflare, can blocked by Anti-Bot modules installed on your origin server. Try disabling any anti-bot modules to prevent your origin from blocking these requests.
```

> Search engine crawlers' requests, when proxied through Cloudflare, can blocked by Anti-Bot modules installed on your origin server. Try disabling any anti-bot modules to prevent your origin from blocking these requests.

Reviewer: ```suggestion
Search engine crawlers' requests, when proxied through Cloudflare, can blocked by anti-bot modules installed on your origin server. Try disabling any anti-bot modules to prevent your origin from blocking these requests.
```
